### PR TITLE
fix(kit): fix filler appearing in `InputDateRange`

### LINKED ISSUE #616: [BUG] InputDateRange the date template disappears
# 🐞 Bug report

### Description

Исчезает шаблон даты в режиме "Другая дата" при наведении курсора, после воода шаблон появляется.

### Versions

Browser [Chrome]
Angular [11]
Package [e.g. @taiga-ui/cdk@2.13.0]

https://user-images.githubusercontent.com/26469604/128844674-7384deb6-0833-4a4f-95ac-e78c9efa2dd9.mov

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [x] Bugfix
-   [ ] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?
before

Closes #616 

## What is the new behavior?
after

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

**Review by waterplea — APPROVED** [review]

File: projects/kit/components/input-date-range/input-date-range.component.ts [hunk 1]
```diff
@@ -312,6 +312,10 @@ export class TuiInputDateRangeComponent
         this.toggle();
         this.focusInput();
 
+        if (!range) {
```
Comment (splincode) [inline projects/kit/components/input-date-range/input-date-range.component.ts:315]: Don't need to add tests for this? ))

**Review by nsbarsukov — DISMISSED** [review]

File: projects/kit/components/input-date-range/input-date-range.component.ts (hunk 1 above)
Comment (MarsiBarsi) [inline projects/kit/components/input-date-range/input-date-range.component.ts:315]: `range` can be only `TuiDayRange`. So, this condition is always false according to typing

**Review by MarsiBarsi — CHANGES_REQUESTED** [review]

File: projects/kit/components/input-date-range/input-date-range.component.ts (hunk 1 above)
Comment (MarsiBarsi) [reply projects/kit/components/input-date-range/input-date-range.component.ts:315]: I mean you might need to fix the type or smth :)

Comment (vladimirpotekhin) [reply projects/kit/components/input-date-range/input-date-range.component.ts:315]: thanks! fixed

Comment (vladimirpotekhin) [reply projects/kit/components/input-date-range/input-date-range.component.ts:315]: I don't think so, this is a change detection problem, the unit test will be useless (

**Review by waterplea — APPROVED** [review]

**Review by splincode — APPROVED** [review]

**Pull request merged**